### ISSUE TITLE
Source code for this procces?

### ISSUE THREAD

**Issue by Nathan22211** [Deutherius/Filament-Width-Comp-Experiments#1]
Been thinking about doing something similar with 3 width sensors, so I've been hoping to use your code to aid in that

**Comment by Deutherius** [comment]
Hey,
not sure what code you mean - I am using the built-in klipper feature for a [hall effect based filament width sensor](https://www.klipper3d.org/Config_Reference.html?h=hall#hall_filament_width_sensor), made no changes to that. Other than that it's just a minor script to extract the filament width callouts from klippy.log, a tiny script to convert .csv files with the values to a graph using plotly and a gihub-pages page to make the graph interactible.

**Comment by Nathan22211** [comment]
Yeah, I was referring to the code used for the filament width compensation. I'm stuck on how you buffered the data till it got to the hotend.

Also, I was going to use Moonraker to get the data from the width sensors.
